lib/bugsnag/report.rb

```diff
@@ -110,7 +116,14 @@ def as_json
       payload_event = Bugsnag::Cleaner.clean_object_encoding(payload_event)
 
       # filter out sensitive values in (and cleanup encodings) metaData
-      payload_event[:metaData] = Bugsnag::Cleaner.new(configuration.meta_data_filters).clean_object(meta_data)
+      filter_cleaner = Bugsnag::Cleaner.new(configuration.meta_data_filters)
+      payload_event[:metaData] = filter_cleaner.clean_object(meta_data)
+      payload_event[:breadcrumbs] = breadcrumbs.map do |raw_crumb|
+        breadcrumb = raw_crumb.to_h
```

Comment:
if `breadcrumbs` is an `Array` of `Breadcrumb` objects, should one of these be `breadcrumb` (instead of `raw_crumb`)? It seems like the variable names should be reversed, as raw implies something more primitive.

Alternatively, the block variable can be `breadcrumb` but the converted one can be named something like `breadcrumb_hash`.